Source/src/WixSharp/Extensions.cs

```diff
@@ -1769,6 +1770,22 @@ static public WixProject IncludeWixExtension(this WixProject project, string ext
             return project;
         }
 
+        /// <summary>
+        /// Adds paths to the assemblies referenced by <see cref="ManagedAction" />s.
+        /// </summary>
+        /// <param name="project">The project.</param>
+        /// <param name="assemblyNames">The assembly names to add.</param>
+        public static void AddDefaultRefAssemblies(this Project project, params string[] assemblyNames)
+        {
+            var referencedAssemblyNames = sys.Assembly.GetExecutingAssembly().GetReferencedAssemblies();
+            var externalAssemblyNames = referencedAssemblyNames
+                .Where(an => assemblyNames.Contains(an.Name));
+            var externalAssemblies = externalAssemblyNames.Select(sys.Assembly.Load);
```

Comment:
Nice use of the method instead of lambda

Reply:
What do you mean?

Reply:
I complimented you on the use of method sys.Assembly.Load instead of lambda asm=> sys.Assembly.Load(asm)